### DISCUSSION TITLE
Issue with cursor alignment

### DISCUSSION THREAD

**Discussion by DONSA** [nvim-tree/nvim-tree.lua#2354] [Q&A]
Do you have any idea why the cursor is not aligned with the arrow?
I have the same settings on another PC and it looks fine.
I've installed a patched font using `brew install font-hack-nerd-font`.

<img width="223" alt="image" src="https://github.com/nvim-tree/nvim-tree.lua/assets/4755300/f6240036-addf-40ed-8d7d-953830422f1d">

**Comment by gegoune** [comment]
It's a font/terminal emulator drawing 'issue'. Nothing nvim-tree can do about that.

**Reply by alex-courtis** [reply]
You might try "Hack Nerd Font Mono" instead of "Hack Nerd Font", althought that is from a linux alacritty user.

**Reply by DONSA** [reply]
Yes, that's what I did in the end. I'm still curious why this happens.
